app/src/main/java/org/mozilla/fenix/settings/PairFragment.kt

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import mozilla.components.feature.qr.QrFeature
+import mozilla.components.support.base.feature.BackHandler
+import mozilla.components.support.base.feature.ViewBoundFeatureWrapper
+import org.mozilla.fenix.BrowserDirection
+import org.mozilla.fenix.HomeActivity
+import org.mozilla.fenix.R
+import org.mozilla.fenix.ext.requireComponents
+
+class PairFragment : Fragment(), BackHandler {
+    private val qrFeature = ViewBoundFeatureWrapper<QrFeature>()
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.fragment_pair, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        (activity as AppCompatActivity).title = getString(R.string.preferences_sync)
```

Comment:
I think if you move this to `onResume` it will get re-set whenever it is re-displayed

Reply:
that works with the back button in the top left corner: 

![image](https://user-images.githubusercontent.com/128755/57335934-33810200-70f2-11e9-8c74-d4517d46f9cf.png)


but not the android back button :(